pkg/util/rand/rand.go

```diff
@@ -81,3 +83,13 @@ func String(length int) string {
 	}
 	return string(b)
 }
+
+// String generates a random hexadecimal string n characters long.  This will
+// panic if n is less than zero.
+func HexString(length int) string {
```

Comment:
Yes, unused indeed, since I switched to crypto/rand. Was going to drop it.

On Thu, 8 Sep 2016, 01:13 Mike Danese, <notifications@github.com> wrote:

> In pkg/util/rand/rand.go
> <https://github.com/kubernetes/kubernetes/pull/31221#discussion_r77927756>
> :
>
> > @@ -81,3 +83,13 @@ func String(length int) string {
> >  	}
> >  	return string(b)
> >  }
> > +
> > +// String generates a random hexadecimal string n characters long.  This will
> > +// panic if n is less than zero.
> > +func HexString(length int) string {
>
> Unused?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/31221/files/6e0fe5561364cdd1a098268ea6af7d78d418ffe4#r77927756>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAPWS0QL9FGenUvbKr-n28U4WJjZd_mDks5qn1M_gaJpZM4JqsTH>
> .
>